### PR TITLE
modification towards the idea of "Separate Dynamic and Static L1 Attributes"

### PR DESCRIPTION
**Pull request by blockchaindevsh**

### CODE REVIEW

File: specs/experimental/inbox-contract.md [hunk 1]
```diff
@@ -55,179 +58,126 @@ These modifications aim to enhance the security and efficiency of the batch subm
...
+- This cached configuration is then utilized by `DataAvailabilitySource.OpenData` to derive batches, as seen [here](https://github.com/ethereum-optimism/optimism/blob/0cf2a11611d329b9c6d53fd6473dbbc44f68e94c/op-node/rollup/derive/l1_retrieval.go#L57).
+
+Post-fork:
+- It consistently reads the system configuration from the L2 state of the parent block.
+
+
+
 ### How `op-node` knows the canonical batch inbox
 
 We define that the canonical batch inbox at a specific L2 block is the batch inbox of `SystemConfig` of the origin of the L2 block.
```
Comment (qizhou) [inline specs/experimental/inbox-contract.md]: Should this part be updated accordingly?

Comment (zhiqiangxu) [reply specs/experimental/inbox-contract.md]: Updated.

Comment (blockchaindevsh) [reply specs/experimental/inbox-contract.md]: Updated.

**Comment by blockchaindevsh** [comment]
Close since moved to upstream.

**Pull request closed without merging**